client/src/pages/Login/LoginPage.module.css

```diff
@@ -1,8 +1,16 @@
+.input-container {
```

Comment:
input bars preview:
<img width="696" alt="image" src="https://github.com/hsu-isaac/fitconnect/assets/53983131/7fea83b0-5a9f-4a54-8fd4-34b908ac43cd">

Reply:
<img width="115" alt="image" src="https://github.com/hsu-isaac/fitconnect/assets/85271794/cdfa5ece-fadf-4a84-9a20-78860945518a">
are these icons just a mac thing or something LOL i have no idea what they are and its not showing up when i run it locally - just wanted to make sure i wasnt missing anything